partcad-cli/src/partcad_cli/click/commands/config.py

```
@@ -0,0 +1,12 @@
+import rich_click as click
+from partcad.logging import Process, info, debug
+from partcad.user_config import user_config
+
+
+@click.command(help="Show the current user configuration")
+@click.pass_obj
+def cli(ctx) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `ctx`

(ANN001)